src/main/java/Controller/newPlaylistController.java

```diff
@@ -0,0 +1,28 @@
+package Controller;
+import InputBoundary.newPlaylistInputBoundary;
+import InputData.newPlaylistInputData;
+
+/**
+ * Controller-layer implementation for create playlist use case
+ */
+public class newPlaylistController {
+    private final newPlaylistInputBoundary newPlaylistInputBoundary;
+    private final newPlaylistInputData newPlaylistInputData;
+    public newPlaylistController(newPlaylistInputBoundary newPlaylistInputBoundary,newPlaylistInputData newPlaylistInputData){
```

Comment:
Again, I was under the impression that the controller should be constructing the use case interactor instead of being given it from the viewModel. if this is incorrect, please let me know.